cli/standalone.rs

```diff
@@ -0,0 +1,114 @@
+use crate::colors;
+use crate::flags::Flags;
+use crate::permissions::Permissions;
+use crate::program_state::ProgramState;
+use crate::tokio_util;
+use crate::worker::MainWorker;
+use deno_core::error::AnyError;
+use deno_core::futures::FutureExt;
+use deno_core::ModuleLoader;
+use deno_core::ModuleSpecifier;
+use deno_core::OpState;
+use std::cell::RefCell;
+use std::convert::TryInto;
+use std::env::current_exe;
+use std::fs::File;
+use std::io::Read;
+use std::io::Seek;
+use std::io::SeekFrom;
+use std::pin::Pin;
+use std::rc::Rc;
+
+pub fn standalone() {
+  let current_exe_path =
+    current_exe().expect("expect current exe path to be known");
```

Comment:
Since `standalone()` is always run I bet there's a high chance of panic happening here, suggest to replace all `expect()` calls with returning result